edx_proctoring/models.py

```diff
@@ -321,6 +322,53 @@ def get_active_student_attempts(self, user_id, course_id=None):
 
         return self.filter(filtered_query).order_by('-created')  # pylint: disable=no-member
 
+    def get_user_onboarding_attempt(self, user_id, course_id=None):
```

Comment:
Can you make this name more descriptive? There's a lot of logic here that is not really clear from the name.

Reply:
Yeah, I was having trouble coming up with something that encompassed all the logic. Also considering your comment below: Maybe something like`get_last_relevant_proctored_practice_attempt`?

Reply:
I think that's better!